### PR TITLE
content: add solution (CSES Planets and Kingdoms)

### PR DESCRIPTION
**Pull request by dongliu0426**

### CODE REVIEW

File: solutions/CSES Planets and Kingdoms [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+/**
+ * Description: Kosaraju's Algorithm, DFS twice to generate 
+ 	* strongly connected components in topological order. $a,b$
+ 	* in same component if both $a\to b$ and $b\to a$ exist.
+ * Time: O(N+M)
+ * Source: Wikipedia
+ * Verification: POI 8 peaceful commission
+ */
+
+struct SCC {
```
Comment (bqi343) [inline solutions/CSES Planets and Kingdoms:34]: indentation looks messed up

File: solutions/CSES Planets and Kingdoms [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    
+    SCC graph;
+    graph.init(n);
+    while(m--) {
+        cin >> a >> b;
+        graph.ae(--a, --b);
+    }
+    graph.gen();
+
+    static int ID[200000], ids;
```
Comment (bqi343) [inline solutions/CSES Planets and Kingdoms:151]: why static?

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/CSES Planets and Kingdoms (hunk 2 above)
Comment (dongliu0426) [reply solutions/CSES Planets and Kingdoms:151]: local variables without static would give undefined values (and i'm too lazy to use memset :P)

Comment (bqi343) [reply solutions/CSES Planets and Kingdoms:151]: just do `int ID[200000]{}`?

File: content/6_Advanced/SCC.mdx [hunk 3]
```diff
@@ -13,7 +13,7 @@ frequency: 1
 
 export const problems = {
 	ex: [
-		new Problem('CSES', 'Planets & Kingdoms', '1683', 'Easy', false, [], ''),
+		new Problem('CSES', 'Planets & Kingdoms', '1683', 'Easy', false, ['SCC'], ''),
```
Comment (bqi343) [inline content/6_Advanced/SCC.mdx:16]: didn't link to the solution

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Comment by bqi343** [comment]
Actually, Planets and Kingdoms should be a focus problem and the solution should go directly in the module.

File: content/6_Advanced/SCC.mdx [hunk 4]
```diff
@@ -77,7 +77,161 @@ export const problems = {
 
 ## SCCs
 
-<Problems problems={problems.ex} />
+<FocusProblem problem={problems.ex[0]} />
+
+## Solution - Planets and Kingdoms
```
Comment (bqi343) [inline content/6_Advanced/SCC.mdx:82]: should go under the tutorials. Also should be `###` so it's under SCC.

File: content/6_Advanced/SCC.mdx [hunk 5]
```diff
@@ -77,7 +77,161 @@ export const problems = {
...
+	void gen() { // fills allComp
+		for(int i=0; i<N; i++) if (!vis[i]) dfs(i);
+		reverse(begin(todo), end(todo)); 
+		for(int x : todo) if (comp[x] == -1) {
+			dfs2(x,x), comps.push_back(x);
+		}
+	}
+};
+int main() {
+
```
Comment (bqi343) [inline content/6_Advanced/SCC.mdx:130]: sort of confused why there's space after but not before the start of the function :eyes:

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Pull request merged**